kitchenSink/requireJSConfig.js

```diff
@@ -24,38 +22,28 @@
       // Set up the Chiasm path.
       // https://github.com/curran/chiasm
       chiasm: chiasmPath + "chiasm",
-  
```

Comment:
This is great - I've been wanting to eliminate the need to list out all the plugins here.